src/codeu/chat/common/ConversationHeader.java

```diff
@@ -63,6 +71,51 @@ public ConversationHeader(Uuid id, Uuid owner, Time creation, String title) {
     this.creation = creation;
     this.title = title;
     this.size = 0;
+    this.permissions = new HashMap<Uuid, Byte>();
+    permissions.put(owner, OWNER_PERM);
+  }
 
+  public boolean isOwner(Uuid user) {
+    if (!(permissions.containsKey(user))) {
+      return false;
+    }
+    Byte currentPermission = permissions.get(user);
+    return ((currentPermission & OWNER_PERM) > 1);
+  }
+
+  public boolean isAdmin(Uuid user) {
+    if (!(permissions.containsKey(user))) {
+      return false;
+    }
+    Byte currentPermission = permissions.get(user);
+    return ((currentPermission & ADMIN_PERM) > 1);
+  }
+
+  public boolean isMember(Uuid user) {
+    if (!(permissions.containsKey(user))) {
+      return false;
+    }
+    Byte currentPermission = permissions.get(user);
+    return ((currentPermission & MEMBER_PERM) > 1);
```

Comment:
this check will return false if the user permissions is only member. 0b0001 & 0b0001 == 1, 1 > 1 is false.